README.md

```diff
@@ -148,3 +148,4 @@ for details.
 ## Credit
 - https://golang.org
 - https://github.com/giantswarm/microkit
+ 
```

Comment:
I think we added this newline for testing earlier.

Reply:
Good catch. Fixed.